### PR TITLE
Extended some rules with suspicious sub processes

### PR DESCRIPTION
**Pull request by Neo23x0**
https://twitter.com/skept1kal/status/1616647571904020481

Publicly reachable applications / services that get exploited spawn a different kind if sub process than local exploited applications. Attackers can use e.g. "net user ... /add" to add a user for a remote login.

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_susp_manageengine_pattern.yml [hunk 1]
```diff
@@ -39,7 +40,15 @@ detection:
...
             - '\notepad.exe'  # Often used in POCs
-    condition: selection
+            - '\systeminfo.exe'
+            - '\net.exe'
+            - '\net1.exe'
+            - '\reg.exe'
+            - '\query.exe'
+    filter1:
+        Image: '\net.exe'
+        Commandline|contains: ' stop'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_manageengine_pattern.yml:50]:
```suggestion
        CommandLine|contains: ' stop'
```

File: rules/windows/process_creation/proc_creation_win_susp_spoolsv_child_processes.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 title: Suspicious Spool Service Child Process
 id: dcdbc940-0bff-46b2-95f3-2d73f848e33b
-status: stable
+status: testing
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_spoolsv_child_processes.yml:3]:
```suggestion
status: test
```

File: rules/windows/process_creation/proc_creation_win_exploit_cve_2020_10189.yml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 title: Exploited CVE-2020-10189 Zoho ManageEngine
 id: 846b866e-2a57-46ee-8e16-85fa92759be7
-status: stable
+status: testing
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_exploit_cve_2020_10189.yml:3]:
```suggestion
status: test
```

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Removed the added `net.exe` from dcdbc940-0bff-46b2-95f3-2d73f848e33b as it was already there in a separate selection called `suspicious_net`. I added `net1.exe` to that too.

**Pull request merged**